src/server.js

```diff
@@ -35,7 +35,8 @@ module.exports = function() {
 		.engine("html", expressHandlebars({
 			extname: ".html",
 			helpers: {
-				tojson: (c) => JSON.stringify(c)
+				tojson: (c) => JSON.stringify(c),
+				translate: (c) => i18n.t(c)
```

Comment:
I'm guessing leave most of them untranslated tokens, add a class like `translate`, and iterate them in client JS on load?